tardis/transport/montecarlo/montecarlo_main_loop.py

```diff
@@ -117,7 +117,7 @@ def montecarlo_main_loop(
     # so we iterate from 0 to n_threads-1 to create the estimator_list
     estimator_list = estimators.create_estimator_list(n_threads)
 
-    for i in prange(no_of_packets):
+    for i in range(no_of_packets):
```

Comment:
This shouldn't be a committed change, just a local test

Reply:
Yes, I will remove that. I was just making sure that it works.